### ISSUE TITLE
Back Office SMTP / Mail crashing the site

### ISSUE THREAD

**Issue by warrentheranga** [umbraco/Umbraco-CMS#10974]
### Which *exact* Umbraco version are you using? For example: 8.13.1 - don't just write v8

8.16.0

### Bug summary

When attempting to create new users via invite, or resending password I get the following error and the crashes the application pool taking down the site. 

![image](https://user-images.githubusercontent.com/5426780/131214411-f8fc5b2c-a878-4e4d-b1c9-71253deb4fe7.png)

I have tried using the same Mailgun SMTP account we have been using (and settings) since this was working back in 8.12 as well as AWS SES creds too - both cause the same issue.

The Umbraco instance is also falling over shortly after opening the Dashboard on sites that have Health Check emails enabled.

### Specifics

Unable to find any logs or output from Umbraco
Can only find the following from Windows event log

![image](https://user-images.githubusercontent.com/5426780/131214534-7aaabb23-a306-4835-9815-aea9dd66ea8f.png)

### Steps to reproduce

Create a new user via
/umbraco/#/users/users/users?invite=true

### Expected result / actual result

Expected for the email to hit the SMTP server but it is not getting that far

**Comment by nul800sebastiaan** [comment]
Are you absolutely sure you're using 8.16.0?  We fixed this specific problem for 8.15.2 https://github.com/umbraco/Umbraco-CMS/issues/10746

And after that it's been merged upwards to 8.16 and will be for future versions as well. I just tested locally and inviting a user works just fine.

![image](https://user-images.githubusercontent.com/304656/131305318-4a223408-3f4a-488b-abf5-7526835d97aa.png)

 I am thinking something might not have upgraded correctly for you maybe?
